src/app/core/entity/entity-config.service.ts

```diff
@@ -144,6 +144,8 @@ export class EntityConfigService {
     entityType.color = entityConfig.color ?? entityType.color;
     entityType.route = entityConfig.route ?? entityType.route;
     entityType.hasPII = entityConfig.hasPII ?? entityType.hasPII;
+    entityType.enableUserAccounts =
```

Comment:
Could you also add a migration function to the ConfigService that
1. adds this flag for every `entity:User` config
2. deletes the panel with UserSecurity component in any "view:user/:id" (but not in other views, we'll handle that manually to simplify the migration logic here)